model-api/src/main/java/ru/training/karaf/model/User.java

```diff
@@ -1,12 +1,16 @@
 package ru.training.karaf.model;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Date;
+import java.util.List;
 import java.util.Set;
 
 public interface User {
-    String getFirstName();
-    String getLastName();
-    String getLogin();
-    Integer getAge();
-    String getAddress();
-    Set<String> getProperties();
+    public UserName getUserName();
```

Comment:
Just for your information: it is not necessary to add public modifier to interface methods - they are public by default